Deploy-AzureOptimizationEngine.ps1

```diff
@@ -143,6 +146,10 @@ if(Test-Path -Path $SilentDeploymentSettingsPath)
             throw "WorkspaceResourceGroupName is required for silent deployment when WorkspaceReuse is set to 'y'."
         }
     }
+    if (-not($deploymentOptions["DeployWorkbooks"]) -or ($deploymentOptions["DeployWorkbooks"] -ne "y" -and $deploymentOptions["DeployWorkbooks"] -ne "n"))
```

Comment:
This does not seem necessary, as we are choosing to deploy (or not) workbooks in a parameter (`-DeployWorkbooks`)